### PR TITLE
Docs/1.0.0 updates

### PR DESCRIPTION
**Pull request by smoreface**
Lots of minor fixes
* Fixing typos
* Fixing broken links

Plus some bigger ones:
* Updating install guidance with more troubleshooting and clarity
* Updating API doc link to point to hosted OpenAPI reference

### CODE REVIEW

File: docs/source/guide/install.md [hunk 1]
```diff
@@ -75,7 +83,7 @@ Start Label Studio:
 docker-compose up -d
 ```
 
-This starts Label Studio with a PostgreSQL database backend. 
+This starts Label Studio with a PostgreSQL database backend and and NGINX proxy web server used to load uploaded data.
```
Comment (niklub) [inline docs/source/guide/install.md:86]: and doubled

File: docs/source/guide/install.md [hunk 2]
```diff
@@ -103,6 +111,39 @@ conda activate label-studio
...
+You might see errors when installing Label Studio. Follow these steps to resolve them.
+
+### Run the latest version of Label Studio
+Many bugs might be fixed in patch releases or maintenance releases. Make sure you're running the latest version of Label Studio by upgrading your installation before you start Label Studio. 
+
+### Errors about missing packages
+
+If you see errors about missing packages, install those packages and try to install Label Studio again. Make sure that you run Label Studio in a clean Python environment, such as a virtual environment. 
+
+For example, you might need to run the following in an Anaconda environment:
```
Comment (niklub) [inline docs/source/guide/install.md:125]: I'm not sure we need to provide the exact package installation commands here: prev paragraph is enough

File: docs/source/guide/install.md [hunk 3]
```diff
@@ -124,13 +165,14 @@ The most important change to be aware of is changes to rename "completions" to "
...
-
 ## Database storage
 Label Studio uses a database to store project data and configuration information. 
 
 ### SQLite database
 
-Label Studio uses SQLite by default. You don't need to configure anything. Label Studio stores all data in a single file in the specified directory of the admin user. After you [start Label Studio](start.html), the directory used is printed in the terminal. 
+Label Studio uses SQLite by default. Label Studio expects version 3.35.x or later of SQLite, so if you have another version installed, you might see unexpected behavior. 
+
+Label Studio stores all data in a single file in the specified directory of the admin user. After you [start Label Studio](start.html), the directory used is printed in the terminal.
```
Comment (niklub) [inline docs/source/guide/install.md:175]: in the specified directory of the admin user -> in the _default application data dir_ (which is different for different OS).
\+ we can also use `LABEL_STUDIO_BASE_DATA_DIR` to override this default

File: docs/source/guide/ml.md [hunk 4]
```diff
@@ -124,7 +124,7 @@ Depending on score types you can select a sampling strategy:
 * prediction-score-min (min is the best score) 
 * prediction-score-max (max is the best score)
  
-See more about active learning sampling in [Set up task sampling for your project](guide/start.html#Set-up-task-sampling-for-your-project). 
+See more about active learning sampling in [Set up task sampling for your project](start.html#Set-up-task-sampling-for-your-project).
```
Comment (niklub) [inline docs/source/guide/ml.md:127]: It's better to put the link here to where we describe prediction score-based filtering/ordering interactions with DM

**Review by niklub — CHANGES_REQUESTED** [review]

File: docs/source/guide/install.md (hunk 1 above)
Comment (smoreface) [reply docs/source/guide/install.md:86]: removed

File: docs/source/guide/ml.md (hunk 4 above)
Comment (smoreface) [reply docs/source/guide/ml.md:127]: Yeah, that still needs to be written but updated the link to the labeling doc.

File: docs/source/guide/export.md [hunk 5]
```diff
@@ -20,11 +20,11 @@ Label Studio supports many common and standard formats for exporting completed l
 
 ### JSON
 
-List of items in [raw JSON format](#raw-json-format-of-completed-labeled-tasks) stored in one JSON file. Use to export both the data and the annotations for a dataset. 
+List of items in [raw JSON format](#Raw-JSON-format-of-completed-labeled-tasks) stored in one JSON file. Use to export both the data and the annotations for a dataset. 
 
 ### JSON_MIN
 
-List of items where only `"from_name", "to_name"` values from the [raw JSON format](#raw-json-format-of-completed-labeled-tasks) are exported. Use to export only the annotations for a dataset. 
+List of items where only `"from_name", "to_name"` values from the [raw JSON format](#Raw-JSON-format-of-completed-labeled-tasks) are exported. Use to export only the annotations for a dataset.
```
[the remark this answers is not in the document]
Comment (smoreface) [reply docs/source/guide/export.md]: Updated

File: docs/source/guide/install.md [hunk 6]
```diff
@@ -16,10 +16,24 @@ Install Label Studio on premises or in the cloud. Choose the install method that
 ## System requirements
 You can install Label Studio on a Linux, Windows, or MacOSX machine running Python 3.6 or later. 
 
-> Note: for Windows users the default installation may fail to build `lxml` package. Consider manually installing it from [the unofficial Windows binaries](https://www.lfd.uci.edu/~gohlke/pythonlibs/#lxml). If you are running Windows 64-bit with Python 3.8 or later, run `pip install lxml‑4.5.0‑cp38‑cp38‑win_amd64.whl`.
+### Install prerequisite
+Install Label Studio in a clean Python environment. You might want to use a virtual environment (venv or conda) to reduce the likelihood of package conflicts or missing packages.
```
[the remark this answers is not in the document]
Comment (smoreface) [reply docs/source/guide/install.md]: Updated

File: docs/source/guide/install.md (hunk 2 above)
Comment (smoreface) [reply docs/source/guide/install.md]: removed specific examples, agreed

File: docs/source/guide/install.md (hunk 3 above)
Comment (smoreface) [reply docs/source/guide/install.md]: updated and added ENVVAR, thanks

File: docs/source/guide/ml.md (hunk 4 above)
Comment (smoreface) [reply docs/source/guide/ml.md]: I'll resolve this more fully with the bigger updates I'll make to this page

**Comment by smoreface** [comment]
@niklub is this one okay to merge too now?

**Review by niklub — APPROVED** [review]

**Pull request merged**